gradle/baseline.gradle

```diff
@@ -17,9 +17,13 @@ List<String> blacklistedBaselineProjects = [
         'profile-client-protobufs',
         'timestamp-impl']
 
+List<String> blacklistedErrorProneProjects = [
```

Comment:
we should definitely whitelist these as there are some critical codepaths we will need logs from.